src/components/atoms/AddToDashboardButton.tsx

```diff
@@ -1,25 +1,23 @@
 import React from 'react';
 import { Button, CrossIcon, PlusIcon } from 'evergreen-ui';
-import { DashboardItemInfo } from '../../types/dashboard';
 
 type AddToDashboardButtonType = {
     onAdd: () => void;
     onRemove: () => void;
-    dashboardItemInfo: DashboardItemInfo;
     added: boolean;
 };
 
 const AddToDashboardButton: React.FC<AddToDashboardButtonType> = (props) => {
     if (props.added) {
         return (
-            <Button appearance="primary" intent="danger" iconBefore={CrossIcon} onClick={props.onRemove}>
+            <Button width="100%" appearance="primary" intent="danger" iconBefore={CrossIcon} onClick={props.onRemove}>
                 Fjern fra ditt dashboard
             </Button>
         );
     }
 
     return (
-        <Button appearance="primary" iconBefore={PlusIcon} onClick={props.onAdd}>
+        <Button width="100%" appearance="primary" iconBefore={PlusIcon} onClick={props.onAdd}>
             Legg til i ditt dashboard
```

Comment:
The text of this button does not fit on my screen. Maybe remove some text or make the button higher?